spec/avro/dsl/test/external_type.rb

```diff
@@ -1,2 +1,2 @@
-namespace :test
+namespace 'test'
```

Comment:
Why was this change necessary? FWIW if I eliminate the `namespace` arg for `eval_file`, then this does not seem necessary.

Reply:
Yep, sorry removed now.